library/src/options.cxx

```diff
@@ -106,57 +106,79 @@ class options::internals
+  this->Internals->init(
+    "window.background.hdri", std::string()); // XXX This overrides window.background.color
+  this->Internals->init("window.background.blur", false);
+
+  this->Internals->init("window.camera.azimuth-angle", 0.0);
+  this->Internals->init("window.camera.elevation-angle", 0.0);
+  this->Internals->init("window.camera.focal-point", std::vector<double>());
+  this->Internals->init("window.camera.position", std::vector<double>());
+  this->Internals->init("window.camera.view-angle", 0.0);
+  this->Internals->init("window.camera.view-up", std::vector<double>());
+
+  this->Internals->init("window.coloring.cells", false);
+  this->Internals->init("window.coloring.name", F3D_RESERVED_STRING);
+  this->Internals->init("window.coloring.component", -1);
+  this->Internals->init("window.coloring.colormap",
     std::vector<double>{
       0.0, 0.0, 0.0, 0.0, 0.4, 0.9, 0.0, 0.0, 0.8, 0.9, 0.9, 0.0, 1.0, 1.0, 1.0, 1.0 });
-  this->Internals->init("component", -1);
-  this->Internals->init("denoise", false);
-  this->Internals->init("depth-peeling", false);
-  this->Internals->init("edges", false);
-  this->Internals->init("filename", false);
-  this->Internals->init("font-file", std::string());
-  this->Internals->init("fps", false);
-  this->Internals->init("fullscreen", false);
-  this->Internals->init("fxaa", false);
-  this->Internals->init("grid", false);
-  this->Internals->init("hdri", std::string());
-  this->Internals->init("inverse", false);
-  this->Internals->init("metadata", false);
-  this->Internals->init("point-sprites", false);
-  this->Internals->init("range", std::vector<double>());
-  this->Internals->init("raytracing", false);
-  this->Internals->init("samples", 5);
-  this->Internals->init("scalars", F3D_RESERVED_STRING);
-  this->Internals->init("ssao", false);
-  this->Internals->init("tone-mapping", false);
-  this->Internals->init("trackball", false);
-  this->Internals->init("volume", false);
-  this->Internals->init("cheatsheet", false);
+  this->Internals->init("window.coloring.range", std::vector<double>());
+
+  this->Internals->init("window.raytracing.enable", false);
+  this->Internals->init("window.raytracing.denoise", false);
+  this->Internals->init("window.raytracing.samples", 5);
+  this->Internals->init("window.actor.bar", false);
+  this->Internals->init("window.actor.edges", false);
+  this->Internals->init("window.actor.filename", false);
+  this->Internals->init("window.actor.fps", false);
+  this->Internals->init("window.actor.grid", false);
+  this->Internals->init("window.actor.cheatsheet", false);
+  this->Internals->init("window.actor.metadata", false);
+
+  this->Internals->init("window.render.depth-peeling", false);
+  this->Internals->init("window.render.fxaa", false);
+  this->Internals->init("window.render.ssao", false);
+  this->Internals->init("window.render.tone-mapping", false);
+
+  this->Internals->init("window.fullscreen", false);
```

Comment:
to remove

Reply:
the option is still present, just no more in the hotkeys